server/services/schema.spec.ts

```diff
@@ -0,0 +1,63 @@
+import '../utils/mockMongo'
+import { createSchema, findSchemaByRef, findSchemasByUse, serializedSchemaFields } from './schema'
+import SchemaModel from '../models/Schema'
+
+describe('test schema service', () => {
+
+  const uploadSchema: any = {
+    name: 'upload-schema',
+    reference: 'upload',
+    use: 'UPLOAD',
+    schema: {}
+  }
+  const uploadSchema2: any = {
+    name: 'upload-schema2',
+    reference: 'upload2',
+    use: 'UPLOAD',
+    schema: {}
+  }
+  const deploymentSchema: any = {
+    name: 'deployment-schema',
+    reference: 'deployment',
+    use: 'DEPLOYMENT',
+    schema: {}
+  }
+
+  beforeEach(async () => {
+    const uploadDoc = new SchemaModel(uploadSchema)
```

Comment:
`await SchemaModel.create(uploadSchema)` is the easier way of doing this.

Reply:
Done